test-app/src/main/AndroidManifest.xml

```diff
@@ -11,6 +11,7 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE_MEDIA_PLAYBACK" />
```

Comment:
Is this required when targeting API 34? Then we need to update the user guide too: https://github.com/readium/kotlin-toolkit/blob/f911a2cae938868d934a4d24ddb1794c3787edf8/docs/guides/media-navigator.md?plain=1#L125

Reply:
Yes it's required for Android 34. Updated the media navigator doc and the migration guide.

Reply:
Thank you!